parbase/FairParRootFileIo.cxx

```diff
@@ -84,7 +84,7 @@ void FairParRootFile::readVersions(FairRtdbRun* currentRun)
     // in the ROOT file
     delete run;
 
-    run = static_cast<FairRtdbRun*>(RootFile->Get((const_cast<char*>(currentRun->GetName()))));
+    run = RootFile->Get<FairRtdbRun>(currentRun->GetName());
```

Comment:
There is no check if the run exist. Should be added.

Reply:
The constructor of `FairParRootFile` sets `run` to nullptr. So this seems to be a valid state for this member variable?

Also this method returns `void` We could put a warning/error?

Reply:
The depends on the definition of valid. Since it is a pointer it shouldn't be nullptr when used. So if we read the object from file we should at lest test it is a valid object afterwards and raise a warning.